Wrappers/Python/ccpi/viewer/ui/dialogs.py

```diff
@@ -69,40 +93,76 @@ def __init__(self, parent, fname):
         fw.addWidget(dimensionalityValue, dimensionalityLabel, "dimensionality")
 
         validator = QtGui.QIntValidator()
-
+        validator.setBottom(0)
         # Entry for size of dimensions:
-        for dim in ['X', 'Y', 'Z']:
+        for dim in ['Width', 'Height', 'Images']:
             ValueEntry = QLineEdit()
             ValueEntry.setValidator(validator)
             ValueEntry.setText("0")
-            Label = QLabel("Size {}".format(dim))
+            Label = QLabel("{}".format(dim))
             fw.addWidget(ValueEntry, Label, 'dim_{}'.format(dim))
 
         dimensionalityValue.currentIndexChanged.connect(self.enableDisableDimZ)
 
         # Data Type
+        self.supported_types = RawInputDialog.supported_types.copy()
         dtypeLabel = QLabel("Data Type")
         dtypeValue = QComboBox()
-        dtypeValue.addItems(["int8", "uint8", "int16", "uint16"])
+        dtypeValue.addItems([dt.name for dt in self.supported_types])
         dtypeValue.setCurrentIndex(1)
         fw.addWidget(dtypeValue, dtypeLabel, 'dtype')
 
         # Endianness
         endiannesLabel = QLabel("Byte Ordering")
         endiannes = QComboBox()
         endiannes.addItems(["Big Endian", "Little Endian"])
-        endiannes.setCurrentIndex(1)
+        endiannes.setCurrentIndex(0)
         fw.addWidget(endiannes, endiannesLabel, 'endianness')
 
         # Fortran Ordering
-        fortranLabel = QLabel("Fortran Ordering")
+        fortranLabel = QLabel("Data Ordering")
         fortranOrder = QComboBox()
-        fortranOrder.addItems(["Fortran Order: XYZ", "C Order: ZYX"])
+        fortranOrder.addItems(["Width-Height-Images", "Images-Height-Width"])
         fortranOrder.setCurrentIndex(0)
         fw.addWidget(fortranOrder, fortranLabel, "is_fortran")
 
+        previewSliceLabel = QLabel("Preview image:")
```

Comment:
Perhaps we should call this "Preview" 

By doing this the reference to the third widget is too strong and it is unclear.

"Slice" would also be appropriate in my opinion, as it can be a slice in the x,y, angle space or any other space used